tools/packaging/packaging.psm1

```diff
@@ -3805,7 +3806,9 @@ function New-MSIPackage
         # Force overwrite of package
         [Switch] $Force,
 
-        [string] $CurrentLocation = (Get-Location)
+        [string] $CurrentLocation = (Get-Location),
+
+        [switch] $LTS
```

Comment:
I had added something with this.  I don't think it's bad for this to be informed about this, even if it doesn't currently use it.